dev_guide/routes.adoc

```diff
@@ -60,7 +61,35 @@ spec:
     name: frontend
 ----
 ====
+endif::[]
+ifdef::openshift-online[]
```

Comment:
Could we used a finer-grained ifdef? It's just the command and yaml that need to be conditionalised.